Presentation.Web/app/components/it-contract/it-contract-overview.controller.ts

```diff
@@ -98,7 +97,8 @@
                 }
             });
 
-            this.activate();
+            //Defer until page change is complete
+            setTimeout(() => this.activate(), 1);
```

Comment:
Hvorfor 1 og ikke 0?

Reply:
For at sikre at den ryger på scheduleren. Implementationen varierer jo efter engine (f.eks. GOOGLE vs andet).. i praksis er det jo næppe målbart